OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -25,7 +25,7 @@ public class WithDecorationInfo : WithDecorationBaseInfo
 		public readonly string Image = null;
 
 		[FieldLoader.Require]
-		[SequenceReference("Image")]
+		[SequenceReference("Image", false, true)]
```

Comment:
Please use `[SequenceReference("Image", actorNameFallback: true)]` here and in the other cases to avoid specifying the redundant `prefix` and make it clearer what is happening.

Reply:
We could perhaps also rename the parameter to `useActorNameIfNull` to make this even more clear?

Reply:
There are 37 other uses of `[SequenceReference("Image")]` in the code. Are we sure that none of them implicitly fall back to the actor name like these ones do?

Reply:
I checked. Only ` RenderVoxels` has a fallback.